### DISCUSSION TITLE
Which type of date format is expected for a blog post?

### DISCUSSION THREAD

**Discussion by shyim** [adityatelange/hugo-PaperMod#456] [Q&A] [answered]
Hey,

I just trying to add a date to my blog post. But always the date does some crazy stuff.

Used: `date: 2017-12-06` page shows `July 0, 60612 `
Sample page date `2020-09-15T11:30:03+00:00` produces `July 30, 15159 ` I have no clue what I am doing there. It can't be that a date is so hard to give a blog post :joy:

**Comment by adityatelange** [comment, marked as answer]
See Hugo docs https://gohugo.io/functions/format/#hugo-date-and-time-templating-reference

Most of the stuff is theme independent,
the ones which are added can be found here https://github.com/adityatelange/hugo-PaperMod/wiki/Variables

**Reply by shyim** [reply]
I have copied this config: https://github.com/adityatelange/hugo-PaperMod/wiki/Installation#sample-configyml
There was
```
DateFormat: "July 4, 2021"
```

After removing that the dates are working as expected :see_no_evil: . Maybe remove that from the example :D

**Reply by adityatelange** [reply]
> I have copied this config: https://github.com/adityatelange/hugo-PaperMod/wiki/Installation#sample-configyml
> There was
> 
> ```
> DateFormat: "July 4, 2021"
> ```
> 
> After removing that the dates are working as expected  . Maybe remove that from the example :D

Ops! Thanks for pointing it out.
